omitting logFilePath results in null pointer exception crash

omitting logFilePath results in null pointer exception crash.
What i was trying to accomplish: Use the new config syntax but only log to stdout/stderr (eg. have systemd/journalctl handle writing the log output to file instead of sync gateway handling its own log files)

Aside: It would also be nice if the systemd unit files stopped redirecting stdout/stderr and instead let systemd handle log capture itself.

### Version info

SG Version: couchbase-sync-gateway-1.4.0-2.x86_64
OS: Centos 7.3.1611

### Config file
relevant config file partial
```javascript
    "logging": {
        "default": {
            "logKeys": ["CRUD", "HTTP"],
            "logLevel": "info"
        }
    }
```

### Log output
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x483d6a]

goroutine 1 [running]:
panic(0xa84cc0, 0xc420010060)
	/usr/local/go/1.7.4/go/src/runtime/panic.go:500 +0x1a1
github.com/couchbase/sync_gateway/base.CreateRollingLogger(0xc4201b44e0)
	/home/couchbase/jenkins/workspace/sgw-unix-build@2/1.4.0/community/godeps/src/github.com/couchbase/sync_gateway/base/logging.go:574 +0x11a
github.com/couchbase/sync_gateway/rest.(*ServerConfig).setupAndValidateLogging(0xc420175680, 0x0, 0xb44bea, 0xe)
	/home/couchbase/jenkins/workspace/sgw-unix-build@2/1.4.0/community/godeps/src/github.com/couchbase/sync_gateway/rest/config.go:482 +0x1ac
github.com/couchbase/sync_gateway/rest.ParseCommandLine(0x401700)
	/home/couchbase/jenkins/workspace/sgw-unix-build@2/1.4.0/community/godeps/src/github.com/couchbase/sync_gateway/rest/config.go:666 +0x985
github.com/couchbase/sync_gateway/rest.ServerMain(0xc400000000)
	/home/couchbase/jenkins/workspace/sgw-unix-build@2/1.4.0/community/godeps/src/github.com/couchbase/sync_gateway/rest/config.go:813 +0x2a
main.main()
	/home/couchbase/jenkins/workspace/sgw-unix-build@2/1.4.0/community/godeps/src/github.com/couchbase/sync_gateway/main.go:34 +0xf5
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x483d6a]
```

### Expected behavior
[Documentation](https://developer.couchbase.com/documentation/mobile/1.4/guides/sync-gateway/config-properties/index.html#default) says without a logFilePath, the server should default to logging to stderr (presumably as long as log rotation is not defined).

### Actual behavior

crash

--- a/base/logging.go
+++ b/base/logging.go
@@ -550,14 +550,16 @@ func NewLoggerWriter(logKey string, serialNumber uint64, req *http.Request) *Log
 
 func CreateRollingLogger(logConfig *LogAppenderConfig) {
 	if logConfig != nil {
+		SetLogLevel(logConfig.LogLevel.sgLevel())
+
 		lj := lumberjack.Logger{}
 
-		if logConfig.LogFilePath != nil {
+		if logConfig.LogFilePath == nil {
+			return
+		} else {
 			lj.Filename = *logConfig.LogFilePath
 			log.Printf("Log entries will be written to the file %v", *logConfig.LogFilePath)
 		}
-
-		SetLogLevel(logConfig.LogLevel.sgLevel())
 
 		if rotation := logConfig.Rotation; rotation != nil {
 			if rotation.MaxSize > 0 {
@@ -571,7 +573,7 @@ func CreateRollingLogger(logConfig *LogAppenderConfig) {
 			}
 			lj.LocalTime = rotation.LocalTime
 		}
-		
+
 		//Update default GoLang logger to use new rolling logger
 		logger.SetOutput(&lj)
 

--- a/base/logging.go
+++ b/base/logging.go
@@ -552,14 +552,14 @@ func CreateRollingLogger(logConfig *LogAppenderConfig) {
 	if logConfig != nil {
 		SetLogLevel(logConfig.LogLevel.sgLevel())
 
-		lj := lumberjack.Logger{}
-
 		if logConfig.LogFilePath == nil {
 			return
-		} else {
-			lj.Filename = *logConfig.LogFilePath
-			log.Printf("Log entries will be written to the file %v", *logConfig.LogFilePath)
 		}
+
+		lj := lumberjack.Logger{}
+
+		lj.Filename = *logConfig.LogFilePath
+		log.Printf("Log entries will be written to the file %v", *logConfig.LogFilePath)
 
 		if rotation := logConfig.Rotation; rotation != nil {
 			if rotation.MaxSize > 0 {

--- a/base/logging.go
+++ b/base/logging.go
@@ -554,6 +554,7 @@ func CreateRollingLogger(logConfig *LogAppenderConfig) {
 
 		if logConfig.LogFilePath != nil {
 			lj.Filename = *logConfig.LogFilePath
+			log.Printf("Log entries will be written to the file %v", *logConfig.LogFilePath)
 		}
 
 		SetLogLevel(logConfig.LogLevel.sgLevel())
@@ -570,8 +571,7 @@ func CreateRollingLogger(logConfig *LogAppenderConfig) {
 			}
 			lj.LocalTime = rotation.LocalTime
 		}
-
-		log.Printf("Log entries will be written to the file %v", *logConfig.LogFilePath)
+		
 		//Update default GoLang logger to use new rolling logger
 		logger.SetOutput(&lj)
 

